example/storybook/stories/AppDemo.stories.tsx

```diff
@@ -1,29 +1,39 @@
 import React, { FC } from 'react';
 import { Text } from 'react-native';
 import { storiesOf } from '@storybook/react-native';
-
 import { ActivityIndicator } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { action } from '@storybook/addon-actions';
 import {
+  ActiveAccountContextProvider,
+  APIClientsContextProvider,
   AuthContextProvider,
   useAuth,
   OAuthLoginButton,
   OAuthContextProvider,
   SettingsScreen,
-} from '../../..';
+} from '../../../src';
```

Comment:
@bruce-glazier I'm finding this to be really useful.  Like we talked about offline, requiring them from an index file like this helps us avoid accidentally forgetting to export in a way that the npm lib will be able to consume.  I'm still in support of the `src/` & `lib/` paths - but just wanted to note we'll want this kind of import in some areas as well.